<?xml version="1.0" encoding="UTF-8"?>
<issues format="6" by="lint 8.6.0-dev" type="baseline" client="" dependencies="true" name="" variant="all" version="8.6.0-dev">

    <issue
        id="CheckResult"
        message="The result of `assertThat` is not used"
        errorLine1="      assertThat(testParse(&quot;&quot;&quot;"
        errorLine2="      ^">
        <location
            file="src/test/java/com/android/aaptcompiler/TableExtractorTest.kt"
            line="266"
            column="7"/>
    </issue>

    <issue
        id="CheckResult"
        message="The result of `assertThat` is not used"
        errorLine1="    assertThat(testParse(&quot;&quot;&quot;"
        errorLine2="    ^">
        <location
            file="src/test/java/com/android/aaptcompiler/TableExtractorTest.kt"
            line="408"
            column="5"/>
    </issue>

    <issue
        id="CheckResult"
        message="The result of `assertThat` is not used"
        errorLine1="    assertThat(testParse(&quot;&quot;&quot;"
        errorLine2="    ^">
        <location
            file="src/test/java/com/android/aaptcompiler/TableExtractorTest.kt"
            line="430"
            column="5"/>
    </issue>

    <issue
        id="SuspiciousIndentation"
        message="Suspicious indentation: This is indented but is not continuing the previous expression (`val extractor =     `...)"
        errorLine1="      try {"
        errorLine2="      ~~~~~">
        <location
            file="src/test/java/com/android/aaptcompiler/TableExtractorTest.kt"
            line="39"
            column="7"/>
        <location
            file="src/test/java/com/android/aaptcompiler/TableExtractorTest.kt"
            line="36"
            column="5"
            message="Previous statement here"/>
    </issue>

</issues>
